demo-plugin/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "demo-plugin",
+    "version": "0.1.0",
+    "private": true,
+    "scripts": {
+        "clean": "rm -rf dist",
+        "build-dev": "yarn clean && NODE_ENV=development yarn ts-node node_modules/.bin/webpack",
+        "start": "yarn clean && NODE_ENV=development yarn ts-node node_modules/.bin/webpack serve",
+        "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'",
+        "format": "yarn prettier -w ."
+    },
+    "devDependencies": {
+        "@openshift-console/dynamic-plugin-sdk": "0.0.15",
+        "@openshift-console/dynamic-plugin-sdk-webpack": "0.0.8",
+        "@patternfly/react-core": "4.162.2",
```

Comment:
```suggestion
```
Core dependency not a dev dep.

Reply:
I am importing Modal from patternfly components as a callback from the ACM custom extension.